app/taskqueue/tasks/report.py

```diff
@@ -236,14 +236,16 @@ def send_multiple_emails_error(
     txt_body, html_body, subject = \
         utils.report.error.create_duplicate_email_report(email_data)
 
+    mail_options = data.get("mail_options")
+
     if (txt_body or html_body) and subject:
         utils.LOG.info(
             "Sending duplicate emails report for %s-%s-%s",
             job, git_branch, kernel)
         utils.emails.send_email(
-            [taskc.app.conf.mail_options.get("error_email")],
+            [mail_options.get("error_email")],
```

Comment:
I'm not convinced this is what we really want to do, nor the correct thing to do.

The idea would be to use the `mail_options` directly taken from the celery app configuration, not passing it around (more or less in the same way we are removing all those `db_options` passing stuff).

The other thing is that when this function is called it doesn't have the `mail_options` data structure.

What you might want to do is fix the `/etc/linaro/kernelci-celery.cfg` config file to reference the `error_email` if it's not in there.

Thinking about this, it might even be useless to define the mail options in the Tornado side, since they are only really used from the Celery app (and we shouldn't pass them along).

Reply:
Well, this was merely to make the existing code actually work...

I see what you mean about moving the email options from the Tornado to the Celery config.  I can send another patch to address that.

Also, maybe the new Jenkins config should also be put into the Celery config (see #24)?  It should also only be used in tasks.

Reply:
I understand, but it would be better to fix the config file in this case since the configuration was coming from there and not from the Tornado options (even if it was passed around and it was missing that variable).

Yeah, better fix it on Celery side and provide defaults there, and probably better to do the same for #24.
Thanks!